src/main/java/com/ecwid/maleorang/method/v3_0/campaign/report/email/GetCampaignEmailActivityForSuscriberMethod.kt

```diff
@@ -0,0 +1,56 @@
+package com.ecwid.maleorang.method.v3_0.campaign.report.email
+
+
+import com.ecwid.maleorang.MailchimpMethod
+import com.ecwid.maleorang.MailchimpObject
+import com.ecwid.maleorang.annotation.*
+import java.util.*
+import org.apache.commons.codec.digest.DigestUtils
+
+/**
+ * Created by: Manuel Lara <lararojas.mr@gmail.com>
+ */
+
+@Method(httpMethod = HttpMethod.GET, version = APIVersion.v3_0, path = " /reports/{campaign_id}/email-activity/{subscriber_hash}")
+class GetCampaignEmailActivityForSuscriberMethod(
```

Comment:
In package `reports/email_activity/` with name `GetEmailActivityForSubscriberMethod.kt`